test.sh

```diff
@@ -127,6 +129,10 @@ make_docker_images () {
     done
 }
 
+# export args to environment so the conditional checks below
+# work as expected.
+export $@
+
```

Comment:
Is this really necessary? When called from `make`, this is already done for you. If calling `test.sh` directly, then you can set it yourself beforehand: `TEST_UNIT=mytest ./test.sh`.